css_tasks/task7/index.html

```diff
@@ -0,0 +1,208 @@
+            </div>
+            <span class="product-description__review-price">$499.00</span>
+          </div>
+        </header>
+        <section class="product-description__info">
+          <header class="product-description__header">
+            <p class="product-description__info-paragraph">
+              Availability :<span>In stock</span>
+            </p>
+            <p class="product-description__info-paragraph">
+              Product Code :<span>#4657</span>
+            </p>
+            <p class="product-description__info-paragraph">
+              Tags :<span>Fashion,</span><span>Hood,</span><span>Classic</span>
+            </p>
+          </header>
+          <p class="product-description__text">
+            Sleek, polished, and boasting an impeccably modern fit, this blue,
+            2-but- ton Lazio suit features a notch lapel, flap pockets, and
+            accompanying flat front trousers—all in pure wool by Vitale Barberis
+            Canonico.
+          </p>
+          <ul class="product-description__text-list">
+            <li class="text-list__item">
+              Dark blue suit for a tone-on-tone look
+            </li>
+            <li class="text-list__item">Regular fit</li>
+            <li class="text-list__item">100% Cotton</li>
+            <li class="text-list__item">Free shipping with 4 days delivery</li>
+          </ul>
+        </section>
+        <form action="" class="product-description__form">
+          <div class="form-inputs">
+            <div class="form-inputs__container form-inputs__container--color">
+              <label for="color">Color</label>
+              <select name="color" id="color" class="form-inputs__color">
+                <option value="">Select Color</option>
+                <option value="black">Black</option>
+                <option value="red">Red</option>
+                <option value="green">Green</option>
+              </select>
+            </div>
+            <div class="form-inputs__container form-inputs__container--size">
+              <label for="size">Size</label>
+              <select name="size" id="size" class="form-inputs__size">
+                <option value="">Select Size</option>
+                <option value="small">S</option>
+                <option value="medium">M</option>
+                <option value="large">L</option>
+                <option value="extra_large">XL</option>
+              </select>
+            </div>
+            <div class="form-inputs__container">
+              <label for="quantity">Qty</label>
+              <input
+                class="form-inputs__quantity"
+                type="number"
+                id="quantity"
+                name="quantity"
+                min="0"
+                max="100"
+                step="1"
+                placeholder="0"
+              />
+            </div>
+          </div>
+          <span class="form-button-clear" role="button">clear selection</span>
```

Comment:
it would be better to use button(probably with type reset) here but not span

Reply:
Done